src/js/components/visualization/ScaleList.jsx

```diff
@@ -29,12 +29,15 @@ var ScaleList = React.createClass({
         </h4>
         <ul>
           {props.scales.map(function(scale) {
+            function updateScaleName(val) {
+              scale.name = val;
+            }
```

Comment:
This approach will make it easy for us to replace these with action dispatcher component properties once scales are in the store